.travis.yml

```diff
@@ -48,9 +48,9 @@ script:
       py.test -s -rxs -v ;
     fi
 
-  - if [[ $TEST_TARGET == 'default' ]]; then
+  - if [[ $TEST_TARGET == 'build_cbi' ]]; then
       python wof/examples/flask/cbi/build_cbi_cache.py ;
-      python wof/examples/flask/swis/runserver_swis.py --config=./test/test_swis_config2.cfg --connection=sqlite:///./test/test_swis2.db ;
+      python wof/examples/flask/swis/runserver_swis.py --config=./test/test_swis_config2.cfg --connection=sqlite:///./test/test_swis2.db > /dev/null & py.test -s -rxs -v ;
```

Comment:
@ocefpaf I don't think this is a part of CBI.

Reply:
I though it used stuff from the step above. I can optimize this in my next commit. Hold on...